File: csharp_courses/gymnasium/sorting_algorithms/Bubblesort.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace InfogandeSortering
{
    class Bubblesort
    {
        public static void BubbleSort(int[] intArray)
        {
            for (int i = intArray.Length - 1; i > 0; i--)
            {
                for (int j = 0; j <= i - 1; j++)
                {
                    if (intArray[j] > intArray[j + 1])
                    {
                        int highValue = intArray[j];
                        intArray[j] = intArray[j + 1];
                        intArray[j + 1] = highValue;
                    }
                }
            }
        }
    }
}

File: java_courses/SortingAlgorithms/src/Bubblesort.java
public class Bubblesort {
    public static void bubbleSort(double[] floatArray){
        for (int i = floatArray.length - 1; i > 0; i--){
            for (int j = 0; j <= i - 1; j++){
                if (floatArray[j] > floatArray[j + 1]){
                    double highValue = floatArray[j];
                    floatArray[j] = floatArray[j + 1];
                    floatArray[j + 1] = highValue;
                }
            }
        }
    }
}
